Atomic/storage.py

```diff
@@ -0,0 +1,52 @@
+import errno
+import shutil
+import selinux
+import requests
+import os, sys
+
+from . import util
+from .Export import export_docker
+from .Import import import_docker
+from .util import NoDockerDaemon
+
+try:
+    from subprocess import DEVNULL  # pylint: disable=no-name-in-module
+except ImportError:
+    DEVNULL = open(os.devnull, 'wb')
+
+try:
+    from . import Atomic
+except ImportError:
+    from atomic import Atomic
+
+class Storage(Atomic):
+    def reset(self):
+        root = "/var/lib/docker"
+        try:
+            self.d.info()
+            raise ValueError("Docker daemon must be stop before resetting storage")
+        except requests.exceptions.ConnectionError as e:
+            pass
+            
+        util.check_call(["docker-storage-setup", "reset"], stdout=DEVNULL)
```

Comment:
This needs to be `--reset` to match https://github.com/projectatomic/docker-storage-setup/pull/128